dashboard/modules/job/job_manager.py

```diff
@@ -418,7 +418,12 @@ def _get_supervisor_runtime_env(
         # Don't set CUDA_VISIBLE_DEVICES for the supervisor actor so the
         # driver can use GPUs if it wants to. This will be removed from
         # the driver's runtime_env so it isn't inherited by tasks & actors.
-        env_vars = runtime_env.get("env_vars", {})
+        # NOTE(edoakes): Can't use .get(, {}) here because we need to handle the case
+        # where env_vars is explicitly set to `None`.
+        env_vars = runtime_env.get("env_vars")
```

Comment:
Concise alternative:

```python
env_vars = runtime_env.get("env_vars") or {}
```

This might be less readable though, so feel free to stick with the current approach.

Reply:
prefer the current one for clarity